contrib/automatic_container_release/BUILD

```diff
@@ -11,8 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
 exports_files(["run_checker.sh.tpl"])
+
+bzl_library(
+    name = "packages_metadata",
+    srcs = [
+        "metadata_merge.bzl",
+        "packages_metadata.bzl",
+    ],
+)
```

Comment:
While you're here, can you add a bzl_library for the other .bzl file in this folder (configs_test)

Reply:
Done